main.py

```diff
@@ -1,127 +1,201 @@
 #!/usr/bin/env python
-import sys
+
+import argparse
+import logging
 import os
+import signal
+import sys
+import threading
+import traceback
+
+from logging.handlers import RotatingFileHandler
 from subprocess import call
 from time import sleep
-import argparse
-#Welcome to pyLCI innards
-#Here, things are about i and o, which are input and output
-#And we output things for debugging, so o goes first.
+
+from apps.manager import AppManager
+from helpers import read_config
+from input import input
 from output import output
+from splash import splash
```

Comment:
On a real device, both PIL and luma.oled/luma.core libraries will be available, however, this is more of a matter of principle - this is not a crucial file, so if a user changes it and ends up being locked out of their ZeroPhone, as opposed to graceful degradation, that's going to be a very stupid situation =)